src/components/DogList.jsx

```diff
@@ -0,0 +1,7 @@
+import './DogList.css';
+
+const DogList = () => {
+  return <h2> Dog List </h2>;
```

Comment:
Let's put a classname on this element (the same as the Dog component does) and update the stylesheet so that it's not styling bare `h2` tags. Since React stylesheets all get merged and aren't intrinsically scoped to a single component, we should be careful about setting styles that could have wider than intended effects.

We _could_ try to make our styles look a little more like the way vite generates them in its template, but we're a bit all over with our naming, even in more student facing materials.

Reply:
Okay, how would you feel about an `id` instead of a `className` since we shouldn't have any other elements like this?

Reply:
I generally try to avoid ids in React other than in places it's required (like to get the `for` attribute to work on form inputs) since nothing prevents a component from being placed on a page multiple times.

Reply:
Ah ok, I will switch it a classname then!

Reply:
Ah okay, I will switch it to a classname then!